config.yaml

```diff
@@ -2,18 +2,21 @@
 
 queue_max_size: 10
 
+log_dir: "logs"
```

Comment:
Rename to `log_directory_path` . Shortening variable names is not necessary and potentially causes confusion.